osu.Game/Rulesets/Mods/ModIncognito.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModIncognito : Mod
+    {
+        public override string Name => "Incognito";
+        public override LocalisableString Description => "Obscure some gameplay visual elements.";
```

Comment:
"Visual" feels redundant here and makes the sentence somewhat irksome in my humble opinion, at least in the way its currently phrased. This is doubly true since the word "obscure" as a verb already invokes a meaning akin to " limited vision " of a given thing. 

"obscure the visuals" is another workable alternative if you want to double down on the visual side, since the fact that its alluding to the gameplay is plenty clear from context alone